server/schedulers/balance_leader.go

```diff
@@ -35,6 +35,10 @@ const (
 	BalanceLeaderType = "balance-leader"
 	// balanceLeaderRetryLimit is the limit to retry schedule for selected source store and target store.
 	balanceLeaderRetryLimit = 10
+	// fitRegionCalling is the limit of the count of calling FitRegion in one scheduler Schedule
+	// when placement rule is enabled.
+	// TODO: set a proper value
+	fitRegionCallingLimit = 10
```

Comment:
The value may cause us cannot find a suitable store.

Reply:
right.

Reply:
Should it be proportional to the number of stores